src/SonarScanner.MSBuild.PreProcessor/WebClientDownloader.cs

```diff
@@ -84,15 +84,21 @@ public string GetHeader(HttpRequestHeader header)
         public async Task<bool> IsLicenseValid(string url)
         {
             this.logger.LogDebug(Resources.MSG_Downloading, url);
-            var response = await this.client.GetAsync(url);
+            var response = await new HttpClient().GetAsync(url);
 
             var content = await response.Content.ReadAsStringAsync();
 
             if (response.StatusCode == HttpStatusCode.NotFound)
             {
                 var json = JObject.Parse(content);
 
-                if (json["errors"] != null)
+                var jsonErrors = json["errors"];
+
+                if (jsonErrors != null &&
```

Comment:
What about this:
if (jsonErrors?.Any(x => x["msg"]?.Value<string>() == "License not found") == true) { .. }

Reply:
Sounds good, thanks for the advice !